<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordMainWindow</class>
 <widget class="QMainWindow" name="RecordMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高清录播直播系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget, QLabel{
                        color:white;
                        border: 1px solid white;
                        padding:0;
                        border-radius:5px;
							background-color:rgb(20,50,70);
                      /**  background-color:rgba(250,170,0,150);**/
  }

/*TabWidget*/
QTabWidget{
/*给tabwidget设置etAttribute(Qt::WA_StyledBackground)*/
border: 1px solid white;
background-color:rgb(20,50,70);
}
QTabWidget::tab-bar{
alignment:left;
left:1px;
}
QTabWidget::pane { 
/*background-color:rgb(20,50,70);*/
border:1px solid white;
} 
QTabBar{
/*只有有按钮的地方才是tabbar，空白处是tabwidget的*/
background-color:white;
}
QTabBar::tab{/*页签*/
min-height:28px;
padding:0 10px;
border:0;
margin:1px 1px 0 0;
background-color:rgb(20,50,70);
}
QTabBar::tab:first{
margin-left:1px;
}
QTabBar::tab:hover{
color:cyan;
}
QTabBar::tab:selected{
background-color: rgb(20, 100, 150);
}
QTabBar::tab:selected:hover{
}

QTabBar::tear{
}
QTabBar::scroller{
}
QMenu {
     background-color: rgb(20,50,70); /* sets background of the menu 设置整个菜单区域的背景色，我用的是白色：white*/
     border: 1px solid white;/*整个菜单区域的边框粗细、样式、颜色*/
 }
  
 QMenu::item {
     /* sets background of menu item. set this to something non-transparent
         if you want menu color and menu item color to be different */
     background-color: transparent;
     padding:8px 32px;/*设置菜单项文字上下和左右的内边距，效果就是菜单中的条目左右上下有了间隔*/
     margin:0px 8px;/*设置菜单项的外边距*/
     border-bottom:1px solid #DBDBDB;/*为菜单项之间添加横线间隔*/
 }
  
 QMenu::item:selected { /* when user selects item using mouse or keyboard */
     background-color: #2dabf9;/*这一句是设置菜单项鼠标经过选中的样式*/
 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>70</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>录制模式</string>
    </property>
    <widget class="QRadioButton" name="m_r_model1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>电影模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_r_model2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>画中画</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_r_model3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>资源模式</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_b_rec">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>360</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>录制开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_pause">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>360</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>录制暂停</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>931</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="vdMain">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>871</width>
       <height>491</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="vd1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>529</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="vd2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>530</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="vd3">
     <property name="geometry">
      <rect>
       <x>749</x>
       <y>529</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>630</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>老师</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>630</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>学生</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>630</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>白板</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="m_zb">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>同步直播</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_res">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>同步录制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>460</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_hh">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>：</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_min">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>：</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_sec">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_zb">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>400</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始直播</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_pg">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>400</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始评估</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_autonav">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>460</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>手动</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_r_teacher">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>490</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>老师</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_r_student">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>490</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_up">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>570</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>上</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_left">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>574</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_down">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>600</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>下</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_right">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>574</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_zoomIn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>650</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>放  大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_zoomOut">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>650</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>缩  小</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
